src/picongpu/include/particles/pusher/particlePusherReducedLandauLifshitz.hpp

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright 2013-2015 Axel Huebl, Heiko Burau, Rene Widera, Richard Pausch
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+#include "types.h"
+#include "simulation_defines.hpp"
+#include "math/Vector.hpp"
+#include "math/RungeKutta.hpp"
+
+
+namespace picongpu
+{
+namespace particlePusherReducedLandauLifshitz
+{
+
```

Comment:
somewhere here the physical method, Marija's paper http://arxiv.org/abs/1502.02432, etc. (as in your PR description) should be referenced